src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
-spring.profiles.active=development
\ No newline at end of file
+spring.profiles.active=development
+# enable lines below for additional debug logging
+#logging.level.org.springframework.web=TRACE
+#logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=DEBUG
+#spring.http.log-request-details=true
+#org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping=TRACE
```

Comment:
These placeholders could go over in application.yml after L22, but not a big deal...however, it always worried me that both application.properties and application.yml are being used.